configfiles/test-large-connected.xml

```diff
@@ -80,5 +92,10 @@
         <probedNListFileName name="prbNListFileName" type="InputFile"/>
       </LayoutFiles>
     </LayoutParams>
+
+    <RecorderParams class="XmlGrowthRecorder" name="RecorderParams">
+      <probedNListFileName name="prbNListFileName" type="InputFile">../configfiles/NList/probedNList_900.xml</probedNListFileName>
```

Comment:
I think this file is for a 900-neuron (30x30) simulation. We can talk about this at our upcoming meeting. I'll not comment on the other config files.